demo/package.yaml

```diff
@@ -0,0 +1,48 @@
+name: myproject
+version: '0.0.0.0'
+description: Please see the README on Github at <https://github.com/abarbu/myproject#readme>
+author: Andrei Barbu
+maintainer: andrei@0xab.com
+copyright: Andrei Barbu 2019
+license: BSD3
+github: abarbu/myproject
+extra-source-files:
+- ChangeLog.md
+- README.md
+dependencies:
+- base >=4.12 && <5
+- godot-haskell
+- linear
+- random
+- deepseq
+- text
+- vector
+- strict-concurrency
+- lens
+- template-haskell
+- th-abstraction
+- containers
+- extra
+library:
+  source-dirs:
+    - src
+    - game
+  exposed-modules:
+  - Lib
+
+verbatim: |
+  foreign-library myproject
+    type: native-shared
+    other-modules: FLib
+    hs-source-dirs: ffi/flib
+    c-sources: ffi/cbits/flib.c
+    include-dirs:
+        /home/andrei/Dropbox/work/godot/godot/modules/gdnative/include/
+      , /home/andrei/Dropbox/work/godot/godot/
+      , /home/andrei/Dropbox/work/godot/godot/platform/x11/
```

Comment:
These includes are a bit problematic

Reply:
Good catch, those should not have been there.